src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -114,15 +116,22 @@ private Cookie generateCustomJwtCookie(final OneLoginUserInfoDto userInfo) {
                 new Cookie(userServiceCookieName, customJwtService.generateToken(claims)),
                 Boolean.TRUE,
                 Boolean.TRUE,
-                null
-        );
+                null);
     }
 
     @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
     private RedirectView getRedirectView(final User user, final Optional<String> redirectUrl) {
-        if(user.isSuperAdmin()) return new RedirectView(adminBaseUrl + "/super-admin/dashboard");
-        if(user.isAdmin()) return new RedirectView(adminBaseUrl + "/dashboard");
+        if (user.isSuperAdmin())
+            return new RedirectView(adminBaseUrl + "/super-admin/dashboard");
+        if (user.isAdmin())
+            return new RedirectView(adminBaseUrl + "/dashboard");
         return new RedirectView((redirectUrl.orElse(configProperties.getDefaultRedirectUrl())));
     }
 
+    @GetMapping("/notice-page")
+    public ModelAndView showNoticePage() {
+        return new ModelAndView(NOTICE_PAGE_VIEW)
+                .addObject("loginUrl", oneLoginBaseUrl);
```

Comment:
This should not be oneLoginBaseUrl.  One login has it's own authorise endpoint that we need to direct users to when they login, not sure if this has been missed out in a previous ticket:

`https://oidc.integration.account.gov.uk/authorize?response_type=code&scope=openid%20email&client_id=dwYu15OryLnH_BGVXUfQI3Uc37U&state=01a4f811-e9bc-41c3-ba3d-81aa1a418cc9&redirect_uri=https%3A%2F%2Fsandbox-gap.service.cabinetoffice.gov.uk%2Fapply%2Fuser%2Fv2%2Fredirect-after-login&nonce=aEwkamaos5C&vtr=%5B%22Cl.Cm%22%5D&ui_locales=en`